SignalMessaging/utils/ConversationStyle.swift

```diff
@@ -227,7 +227,10 @@ public class ConversationStyle: NSObject {
 
     @objc
     public func quotedReplyBubbleColor(isIncoming: Bool) -> UIColor {
-        if isIncoming {
+        if Theme.isDarkThemeEnabled {
+            let alpha: CGFloat = (isIncoming ?0.6 :0.5)
```

Comment:
format nit: missing space after `?`

Reply:
Done. I don't know why clang-format doesn't work better.  When we get a chance, I'd be up for switching formatters.

Reply:
clang-format doesn't format swift code, does it?